cluster/service/cluster_service_blackbox_test.go

```diff
@@ -630,6 +631,54 @@ func (s *ClusterServiceTestSuite) TestListForAuth() {
 	})
 }
 
+func (s *ClusterServiceTestSuite) TestDelete() {
+
+	s.T().Run("ok", func(t *testing.T) {
+		// given
+		c := test.CreateCluster(t, s.DB)
+		sa := &authtestsupport.Identity{
+			Username: auth.ToolChainOperator,
+			ID:       uuid.NewV4(),
+		}
+		ctx, err := authtestsupport.EmbedServiceAccountTokenInContext(context.Background(), sa)
+		require.NoError(t, err)
+		// when
+		err = s.Application.ClusterService().Delete(ctx, c.ClusterID)
+		// then
+		require.NoError(t, err)
+		// check that
+		c, err = repository.NewClusterRepository(s.DB).Load(ctx, c.ClusterID)
+		require.Error(t, err)
+		assert.IsType(t, errors.NotFoundError{}, err)
```

Comment:
`testsupport.AssertError()` can be used instead of `require.Error(t, err)` and `assert.IsType()`

Reply:
ok, done in 870041f